src/ipyaladin/aladin.py

```diff
@@ -336,6 +375,63 @@ def add_table(self, table: Union[QTable, Table], **table_options: any) -> None:
             buffers=[table_bytes.getvalue()],
         )
 
+    def add_overlay(
+        self,
+        region: SupportedRegion,
+        **graphic_options: any,
+    ) -> None:
+        """Add an overlay layer to the Aladin Lite widget.
+
+        Parameters
+        ----------
+        region: str, `~CircleSkyRegion`, `~EllipseSkyRegion`, `~LineSkyRegion`,
+         `~PolygonSkyRegion`, `~RectangleSkyRegion`
+            The region to add in Aladin Lite. It can be given
+            as a string or one of the supported regions
+        graphic_options: keyword arguments
+            The options for the graphic overlay. Use Region visual for region options.
```

Comment:
Add links to Aladin Lide options and to region visual options

Reply:
We only link some of regions.Visual keys to Aladin Lite shape options, how to explain it correctly ?

Reply:
It'd be best to write the list of supported regions.Visual keys explicitely